apps/admin-gui/src/app/shared/components/dialogs/add-member-dialog/add-member-dialog.component.html

```diff
@@ -1,80 +1,105 @@
+          (click)="add.emit()"
+          [disabled]="selection.selected.length === 0 || !addAuth">
+          {{'DIALOGS.ADD_MEMBERS.CREATE' | translate}}
+        </button>
+      </span>
     </div>
   </div>
-  <mat-spinner class="mr-auto ml-auto" *ngIf="processing"></mat-spinner>
-  <div mat-dialog-actions *ngIf="!processing">
-    <button mat-flat-button class="ml-auto" (click)="onCancel()">
-      {{'DIALOGS.ADD_MEMBERS.CANCEL' | translate}}
-    </button>
-    <span
-      matTooltip="{{'DIALOGS.ADD_MEMBERS.ADD_MEMBER_PERMISSION_TOOLTIP' | translate}}"
-      matTooltipPosition="above"
-      [matTooltipDisabled]="selection.selected.length === 0 || this.data.type !== 'group' || (list !== undefined && list.addAuth)">
-      <button
-        *ngIf="this.data.type !== 'group' || (this.data.type === 'group' && inviteAuth)"
-        [disabled]="selection.selected.length === 0 || loading"
-        [matMenuTriggerFor]="menu"
-        class="ml-2 dropdown-toggle"
-        color="accent"
-        mat-flat-button>
-        {{'DIALOGS.ADD_MEMBERS.INVITE' | translate}}
-      </button>
-      <mat-menu #menu="matMenu">
-        <button (click)="onInvite(lang)" *ngFor="let lang of languages" mat-menu-item>
-          {{'DIALOGS.ADD_MEMBERS.INVITE_IN_LANGUAGE' | translate}}
-          {{'SHARED_LIB.LANGUAGES.'+lang | uppercase | translate}}
-          {{'DIALOGS.ADD_MEMBERS.LANGUAGE' | translate}}
+  <div *ngIf="failed.length !== 0">
+    <mat-spinner *ngIf="loading" class="mr-auto ml-auto"></mat-spinner>
+    <div mat-dialog-content *ngIf="!loading">
+      <perun-web-apps-alert
+        class="mb-2"
+        [alert_type]="'error'"
+        >{{'DIALOGS.ADD_MEMBERS.FAILED_DESC' | translate}}</perun-web-apps-alert
+      >
+      <table [dataSource]="failedCandidateDataSource" class="w-100" mat-table>
+        <ng-container matColumnDef="userName">
+          <th *matHeaderCellDef mat-header-cell>
+            {{'DIALOGS.ADD_MEMBERS.NAME_COLUMN' | translate}}
+          </th>
+          <td *matCellDef="let failedCandidate" mat-cell>
+            {{(failedCandidate.candidate.candidate ?? failedCandidate.candidate.richUser) | userFullName}}
+          </td>
+        </ng-container>
+        <ng-container matColumnDef="error">
+          <th *matHeaderCellDef mat-header-cell>
+            {{'DIALOGS.ADD_MEMBERS.ERROR_COLUMN' | translate}}
+          </th>
+          <td *matCellDef="let failedCandidate" mat-cell>
+            {{failedCandidate.errorMsg}}
+            <div class="text-muted">
+              {{failedCandidate.errorName}}
+            </div>
+          </td>
+        </ng-container>
+        <tr *matHeaderRowDef="['userName', 'error']" mat-header-row></tr>
+        <tr *matRowDef="let failedCandidate; columns: ['userName', 'error'];" mat-row></tr>
+      </table>
+      <div mat-dialog-actions *ngIf="!loading">
+        <button mat-flat-button class="ml-auto" (click)="cancel.emit()">
+          {{'DIALOGS.ADD_MEMBERS.CLOSE' | translate}}
```

Comment:
when user click on this button, the table in the vo/group-members page don't refresh - but In this situation some members might be added but user don't see the added members